documentation/agent/cron.md

```diff
@@ -1,49 +0,0 @@
----
```

Comment:
I don't agree to remove the cron page. The page still have a link against usage page and I would prefer to remove all "Cron mode" occurrences explaining this is a half-managed mode, maybe by just replacing them by "Cron based half-managed mode".